buildpipeline/pipeline.json

```diff
@@ -98,7 +98,7 @@
             "PB_DistroRid": "ubuntu.14.04-arm", 
             "PB_DockerTag": "ubuntu-14.04-cross-0cd4667-20172211042239", 
             "PB_TargetArchitecture": "arm", 
-            "PB_AdditionalBuildArguments":"-TargetArchitecture=arm -DistroRid=linux-arm -DisableCrossgen=true -PortableBuild=true -SkipTests=true -CrossBuild=true",
+            "PB_AdditionalBuildArguments":"-TargetArchitecture=arm -DistroRid=linux-arm -DisableCrossgen=true -PortableBuild=true -SkipTests=true",
```

Comment:
I duplicated the change from the commit you referenced - https://github.com/dotnet/core-setup/commit/64a0b92cb524427a620f9760a4f02b41444eb089. Should I add -CrossBuild=true back in?